.travis.yml

```diff
@@ -12,21 +12,24 @@ services:
 before_install:
   - sudo /etc/init.d/mysql stop
   - sudo apt-get update
-  - pushd db
-  - docker build -t faf-db .
-  - docker run -d --name faf-db -e MYSQL_ROOT_PASSWORD=banana -e DB_NAME=faf -p 3306:3306 faf-db
-  - until docker exec -i faf-db ./healthcheck.sh 2>/dev/null; do sleep 1; done
-  - docker exec -i faf-db mysql -uroot -pbanana faf < test-data.sql
+  - git clone https://github.com/FAForever/faf-stack.git faf-stack
+      && pushd faf-stack
+      && git checkout 2873b47
+      && cp -r config.template config
+      && ./scripts/init-db.sh
+      && wget https://raw.githubusercontent.com/FAForever/db/`grep -oP 'faforever/faf-db-migrations:\K(.*)$' ./docker-compose.yml`/test-data.sql
```

Comment:
I know you copied it from me, but maybe we should use $(command) over \`command`